src/api/libcellml/referencerules.h

```diff
@@ -19,14 +19,15 @@ limitations under the License.
 namespace libcellml {
 
 /**
- * @brief The SpecificationRule enum class.
+ * @brief The ReferenceRule enum class.
  *
  * Enum describing the specification rules that are used
```

Comment:
this is no longer specific to the specification, right?

Reply:
I think it will be a super-set of the specification, so the current chapter reference numbers with others associated with code generation (and whatever else) on top.

Reply:
So you are agreeing that the documentation is inaccurate?